src/include/compat.h

```diff
@@ -79,9 +79,16 @@
 /* get PATH_MAX */
 #include <limits.h>
 
+/* Linux error constarts: see http://man7.org/linux/man-pages/man3/errno.3.html */
```

Comment:
please drop this comment. see https://github.com/ceph/ceph/pull/25302#discussion_r